docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -1,49 +1,58 @@
 [[run-digger-installer]]
-== Run Digger Installer
+=== Run Digger Installer
 
-.To run the Digger installer, follow these steps:
 
-. Clone the https://github.com/aerogear/digger-installer[Digger Installer GitHub repo^]
+. Clone the https://github.com/aerogear/aerogear-digger-installer[Digger Installer GitHub repo^]
+
+. Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure. https://github.com/aerogear/aerogear-digger-installer/blob/master/inventory-sample[Sample Ansible Inventory file^].
 
-. Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure
 +
-NOTE: For a full list of variables that can be configured via the inventory file see the link:#installer-variables[Installer Variables] section.
+NOTE: For a full list of variables that can be configured via the inventory file refer to <<installer-variables>>
 
-. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal.:
+. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal:
++
+NOTE: Review https://github.com/aerogear/aerogear-digger-installer/blob/master/sample-build-playbook.yml[Digger Playbook^].
 
-** Command to install digger without a macOS server
+** Command to install Digger without a macOS server
 +
 ----
-ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=osx-provision
+ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=provision-osx
 ----
 
-** Command to install digger with a macOS server
+** Command to install Digger with a macOS server
 +
 ----
 ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml
 ----
 
-. Carry out manual steps during the installation
+** Roles can be run individually or in batch with flag --tags=<role_tag_1>,...,<role_tag_x>
++
+----
+ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --tags=provision-osx,nagios
+----
+** To skip default roles, similarly to --skip-tags=provision-osx e.g. --skip-tags=<role_tag_1>,...,<role_tag_x>
 +
-There are three points during the installation where there are manual steps required.
+----
+ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=provision-osx,nagios,java
+----
+
+
+. Manually accept Android SDK licence
 +
-.. Accepting the Android SDK Licence
-.. Copy the server's (where ansible task are running) public ssh key to Jenkins
-.. Accept the Oracle Licence when installing Java on macOS server
```

Comment:
I think we may still need this @aliok?

Reply:
We still need "-.. Accept the Oracle Licence when installing Java on macOS server"

Reply:
@laurafitzgerald @odra says that yes, we still need this
re ssh : copy ssh public key to jenkins slave on macos

todo

Reply:
resolved